docs/development.md

```diff
@@ -21,7 +21,8 @@ cd .../hmcts-common-platform-mock-api
 - create dummy data
 ```
 # create some data
-$ rails console
+rails mock:demodata:load
```

Comment:
@mabely I think if we document to run rails` mock:demodata:load` we do not need ```
rails console
Pry> FactoryBot.create(:realistic_prosecution_case, defendants: FactoryBot.create_list(:realistic_defendant, 3))
``` (as I think these would do the same thing).

Reply:
Removed it - thanks for spotting it!